lib/tasks/vtul/seamus_importer.rb

```diff
@@ -0,0 +1,221 @@
+      performance.subject = item.metadata.instrumentation
+      performance.subject = ["(none)"] if performance.subject.empty?
+      performance.description = [item.body, item.metadata.subtitle]
+      performance.duration = item.metadata.duration
+
+      # For now, all provided SEAMUS items appear to be public
+      performance.visibility = Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_PUBLIC
+
+      performance.save!
+      puts "... Performance Created Successfully." + performance.inspect
+      create_performance_derivatives(item)
+      performance
+    end
+
+    def create_composition_derivatives(item)
+      puts "... Creating Composition Derivatives [TODO LIBTD-1317]"
+    end
+
+    def create_performance_derivatives(item)
+      puts "... Creating Performance Derivatives [TODO LIBTD-1317]"
+    end
+
+    def get_first_class_src(page, class_name)
+      page.css(class_name).css('img').first['src'] unless page.css(class_name).css('img').first.nil?
+    end
+
+    def get_first_class_href(page, class_name) 
+      page.css(class_name).css('a').first['href'] unless page.css(class_name).css('a').first.nil?
+    end
+
+    def personal_statement(profile)
+      if profile.member_role.empty?
+        profile.short_bio.to_s
+      else 
+        (profile.member_role.to_s + ". " + profile.short_bio.to_s)
+      end
+    end
+
+    def google_plus_handle(uri)
+      # Google plus handles seem more complicated to parse.
+      # Assumes that the uri is already known to be a google plus uri
+      path_fragments = uri.path.split("/").reject{ |s| s.empty? }
+      if path_fragments.size < 1
+        return nil
+      elsif path_fragments.first[0] == '+'
+        return path_fragments.first[1..-1] #remove the '+'
+      else
+        return nil # unable to parse
+      end
+    end
+
+    def handle(link)
+      return link if link.nil?
+
+      link_uri = URI(link)
+      if (link_uri.host == "plus.google.com") || (link_uri.host == "google.com")
+        return google_plus_handle(link_uri)
+      else
+        # The following seems to work for twitter and facebook.
+        # Returns the first part of the path as the handle
+        # Otherwise, returns nil
+        return link_uri.path.split("/").reject{ |s| s.empty? }.first
+      end
+    end
+
+    def create_user(wp_author, user_profile)
+      puts "... Attempting to create new COMPEL user: " + wp_author.email
+
+      user = User.new({email: wp_author.email})
+      user.display_name = wp_author.display_name
+      user.password = SecureRandom.uuid
+      user.personal_statement = personal_statement(user_profile)
+      user.twitter_handle = handle(user_profile.member_twitter_link)
+      user.facebook_handle = handle(user_profile.member_facebook_link)
+      user.googleplus_handle = handle(user_profile.member_google_link)
+        
+      if !( user_profile.member_website_link.nil? || user_profile.member_website_link.blank?)
```

Comment:
Technically, I think `empty?` maybe fail if it's `nil`. 

```
irb(main):001:0> x = nil
=> nil
irb(main):002:0> x.empty?
NoMethodError: undefined method `empty?' for nil:NilClass
	from (irb):2
```

The tough part is that this whole rake task isn't the cleanest. I don't know if the member website link is ever nil. It might always be set to "".